packages/thirdweb/src/indexer/endpoints/getContractTransactions.ts

```diff
@@ -0,0 +1,120 @@
+import type { Transaction } from "viem";
+import type { ThirdwebContract } from "../../contract/contract.js";
+import { getClientFetch } from "../../utils/fetch.js";
+import type { Prettify } from "../../utils/type-utils.js";
+import { formatIndexerTransactions } from "../formatter.js";
+import { addRequestPagination } from "../paging.js";
+import type {
+  IndexerInternalTransaction,
+  IndexerPagingParams,
+  IndexerResponse,
+} from "../types.ts";
+import { getTransactionsEndpoint } from "../urls.js";
+
+export type GetContractTransactionsParams = Prettify<
+  {
+    /**
+     * Contract fetch transactions for
+     */
+    contract: ThirdwebContract;
+    /**
+     * Start of the date range to search in. Default is 1 day ago
+     */
+    startDate?: Date;
+    /**
+     * End of the date range to search in. Default is current date
+     */
+    endDate?: Date;
+  } & IndexerPagingParams
+>;
+
+export type GetTransactionsResult = {
+  transactions: Transaction[];
+  page?: number;
```

Comment:
am I not inputting the page & limit or offset etc (the pagination params) into the call? so wouldn't I already *know* what page I'm getting? I think this should be removed and we can just return `Transaction[]` directly?